### PR TITLE
Ytb 39 delete blogpost

### PR DESCRIPTION
**Pull request by sennikrauss**

### CODE REVIEW

**Review by nforgerit — CHANGES_REQUESTED** [review]
Hey! 

* I might be missing sth but I think this PR is missing the db migration. Please add one to [this directory](https://github.com/idealo/nwp/tree/master/packages/idealo.design/src/server/db_Configuration), and prefix it with "2-". Then your peers don't need to figure out what to change in the db. Thanks!
* Please use a [Transaction](https://www.postgresql.org/docs/8.3/tutorial-transactions.html) for the prev/next blogpost db changes. The node process might crash in between or several people do the same at the same time and then we end up with inconsistent data in production.
* Please make use of the proper column type for boolean values (instead of using string and 'f').

If you have any questions, please don't hesitate to ask! =)

Best, Nico

**Review by MuellerBettina — CHANGES_REQUESTED** [review]
Great code! 

Could you please change these Button tags to lowercase html button tags? 

```
<Button onClick={props.onCancel}>Close</Button>
                      <Button onClick={props.onArchive}>Archive</Button>
                      <Button onClick={props.onDelete}>Delete</Button>
                  </div>:
                  <div>
                      <Button onClick={props.onHide}>Close</Button>
                      <Button onClick={props.onLeave}>Yes</Button>
```
and respectively remove the Button import? Since we'll hopefully be using our own button-Module in the future. 

This is also very likely going to solve the merge conflict ;-)

**Comment by nforgerit** [comment]
> Could you please change these Button tags to lowercase html button tags?

@MuellerBettina I think this won't work, would it? E.g. onClick is a react-ism which is applicable to React components. `<button>` is just a plain html tag. But I might've missed sth.

File: packages/idealo.design/src/server/index.js [hunk 1]
```diff
@@ -199,6 +201,18 @@ app.put('/api/blogposts', isAuthenticated, async (req, res) => {
   return res.json(createdBlogpost);
 });
 
+app.delete('/api/blogposts/delete', isAuthenticated, async (req,res) => {
```
Comment (nforgerit) [inline packages/idealo.design/src/server/index.js:204]: Sorry for the nitpicking but I'd like to keep the API as RESTful as possible. In this case, it means to make the endpoint look like:
`DELETE /api/blogposts/:blogpost_id`. However, I like that you're already using http DELETE.

File: packages/idealo.design/src/ui/pages/BlogPage/DetailView.jsx [hunk 2]
```diff
@@ -138,8 +155,16 @@ const BlogDetailView = (props) => {
                                   <span>Next</span>
                                 </Link>)}
       </div>
+
+      <Prompt
+          show = {prompt.isPromptOpen}
```
Comment (nforgerit) [inline packages/idealo.design/src/ui/pages/BlogPage/DetailView.jsx:160]: Please remove the spaces around the `=` to keep the formatting the same as everywhere else.

**Comment by MuellerBettina** [comment]
> > Could you please change these Button tags to lowercase html button tags?
> 
> @MuellerBettina I think this won't work, would it? E.g. onClick is a react-ism which is applicable to React components. `<button>` is just a plain html tag. But I might've missed sth.

@nforgerit It does work! 😉 (Also you wanted me to changed it before, remember?)

**Review by nforgerit — APPROVED** [review]
Code is fine for me so far. Please finish merging the testing branch and make sure there are no defects. Thank you & apologies it took me so long to give my OK for this PR.

**Review by MuellerBettina — APPROVED** [review]

**Pull request merged**